service.smartthings/addon.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.smartthings" version="1.0.0" name="Smartthings Monitor" provider-name="Lunatixz">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+        <import addon="script.module.requests" version="2.7.0"/>
```

Comment:
You're not required to change this but jfyi you're too far from the most recent requests version available in the repository (this only specifies the minimum version so no problem for me). I recently had some problems in one of my addons because I was (somehow) using a very outdated version of requests

Reply:
Good point, THX... I get in the habit of using older version numbers. I should seek out relatively newer versions.